app/scripts/controllers/UsernameModalInstanceCtrl.js

```diff
@@ -10,7 +10,7 @@
       if (this.username === undefined) {
         alert("You didn't enter a username!");
       } else {
-          ApiRequests.create_user(this.username, this.password);
+          UsersApiRequests.create_user(this.username, this.password);
```

Comment:
`createUser`

Reply:
You could also refactor `create_user` to accept one argument, a user object containing username and password:

```js
{
    "username": username,
    "password": password
}
```

Which would be cleaner.